eng/Versions.props

```diff
@@ -50,8 +50,6 @@
     <UsingToolNetFrameworkReferenceAssemblies Condition="'$(OS)' != 'Windows_NT'">true</UsingToolNetFrameworkReferenceAssemblies>
     <!-- Disable XLIFF tasks -->
     <UsingToolXliff>false</UsingToolXliff>
-    <!-- Use custom version of Roslyn Compiler -->
-    <UsingToolMicrosoftNetCompilers>true</UsingToolMicrosoftNetCompilers>
```

Comment:
This means the build will _not_ get `$(MicrosoftNetCompilersToolsetVersion)` or the Microsoft.Net.Compilers.Toolset reference from Arcade. Instead, we'll pick up the compiler from the .NET SDK. Any idea if that's new enough❔